bin/conjur-cli.rb

```diff
@@ -66,6 +66,9 @@ def test_select
       system "rake account:create[#{account}]" or exit $?.exitstatus
     end
 
+    puts "Validating Conjur data key..."
```

Comment:
Conjur? @rafis3  please confirm that this is okay? Should it be product (DAP/Conjur) specific, depending on the context?

Reply:
Conjur is a component rather than a product. DAP contains the Conjur component. So to me, specifying that it's a Conjur data key is completely fine. But we should be consistent in the other locations where we reference it.